perceval/backends/gerrit.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Alvaro del Castillo San Felix <acs@bitergia.com>
+#
+
+'''Gerrit backend for Perseval'''
```

Comment:
El lun, 21-12-2015 a las 08:48 -0800, Alvaro del Castillo escribió:
> In perceval/backends/gerrit.py:
> > +# (at your option) any later version.
> > +#
> > +# This program is distributed in the hope that it will be useful,
> > +# but WITHOUT ANY WARRANTY; without even the implied warranty of
> > +# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
> > +# GNU General Public License for more details.
> > +#
> > +# You should have received a copy of the GNU General Public
> License
> > +# along with this program; if not, write to the Free Software
> > +# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-
> 1307, USA.
> > +#
> > +# Authors:
> > +#   Alvaro del Castillo San Felix <acs@bitergia.com>
> > +#
> > +
> > +'''Gerrit backend for Perseval'''
> Converted all to use double quotes. About the doc, I have updated all
> "bugs" occurrences to "reviews.. And about the classes and public
> methods doc, I am not sure we need to copy and paste the same doc
> when documenting the public backend API, for example. What do you
> think? I can copy and paste bugzilla doc if you like this approach!
> (and adapt it if needed, but if it is backend API it should be
> exactly the same).
> I will review other public API in the backend, it should be strange
> to have this kind of API, in order to document it!
No, I meant you have to include documentation saying which the
attributes to initialize the Gerrit backend are needed or what the
fetch method does, etc.
> 
> 
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> 
> 
>   > 
>   > 

> 
> 

>